build/teamcity/cockroach/ci/builds/build_impl.sh

```diff
@@ -24,4 +24,5 @@ bazel build //pkg/cmd/bazci --config=ci
 $(bazel info bazel-bin --config=ci)/pkg/cmd/bazci/bazci_/bazci --compilation_mode opt \
 		       --config "$CONFIG" --config ci --config with_ui \
 		       build //pkg/cmd/cockroach-short //pkg/cmd/cockroach \
+               //pkg/cmd/cockroach-sql \
```

Comment:
Nit: indentation looks inconsistent here.

Reply:
Yeah.... the surrounding lines are a mix of tabs and spaces. The added one is spaces only. I didn't want to blindly copy.

Reply:
Ah, OK. Next time you notice this you can feel free to adjust the indentation for the entire file. Emacs sometimes gives me tabs in shell scripts, IDK.